### ISSUE TITLE
Win<bug> 

### ISSUE THREAD

**Issue by Eaphyr** [manucabral/YoutubeMusicRPC#48]
**Describe the bug**
WinError 2 issue
**To Reproduce**
Steps to reproduce the behavior.
start the .exe
**Expected behavior**
I expected no erros
**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. Windows]
 - Version [e.g. Pro 64 bits]
Windows 11 Pro 64 bits
**Additional context**
[client.log](https://github.com/user-attachments/files/16195302/client.log)

**Comment by Eaphyr** [comment]
The title was supposed to be ''WinError 2 <bug>'', my bad

**Comment by manucabral** [comment]
Hi,
This problem is due to the system's language according to my diagnosis. Please try using Windows in English. If this is not the case, please let me know more details.
Thanks for reporting it.

**Comment by Eaphyr** [comment]
Oh alright, i'll change the language that's fine to me. I'll reply again if i come across the issue again in English

**Comment by Eaphyr** [comment]
Yep, sadly, the same issue; the same WinError2, just in English. I'm aware that Windows still retains some of the old language it was installed on, but i'm not sure if that's the issue anymore.

**Comment by manucabral** [comment]
Please try the [v0.5.1](https://github.com/manucabral/YoutubeMusicRPC/releases/tag/0.5.1) version. maybe it will work for you as an alternative.

**Comment by Eaphyr** [comment]
Sadly that doesn't work aswell

**Comment by alexthe-canadian** [comment]
Try using a different browser. If you still get the error then let us know.
